.github/workflows/release-generate-ci.yaml

```diff
@@ -51,12 +54,17 @@ jobs:
       - name: Generate CI (on workflow dispatch)
         if: github.event_name == 'workflow_dispatch'
         working-directory: ./src/github.com/openshift-knative/hack
-        run: go run ./cmd/prowcopy --branch ${{ inputs.branch }} --tag ${{ inputs.branch }} --config config/serverless-operator.yaml
+        run: go run ./cmd/prowcopy --branch ${{ inputs.branch }} --tag ${{ inputs.tag }} --config config/serverless-operator.yaml
```

Comment:
Why not call it with:

```
go run github.com/openshift-knative/hack/cmd/prowcopy
```

(or with `@main` if we don't keep track of the `github.com/openshift-knative/hack/cmd/prowcopy` in go.mod)